src/services/data/handlers/sinkMain.ts

```diff
@@ -198,7 +198,7 @@ const onemac = async (kafkaRecords: KafkaRecord[], topicPartition: string) => {
               // Handle the appk children when an appk parent id is updated
               // I'd like a better way to identify an appk parent.
               if (
-                item._source.authority === Authority["1915c"] &&
+                item._source.authority === AUTHORITY["1915(c)"] &&
```

Comment:
If we type out `_source.authority` to `Authority`, we can remove this constant and just check:

```ts
item._source.authority === "1915(c)"
```

This isn't possible with enums